Dockerfile.win10.min

```diff
@@ -125,7 +125,7 @@ WORKDIR /
 #
 # Installing Vcpkg
 #
-ARG VCPGK_VERSION=2024.03.19
+ARG VCPGK_VERSION=2024.06.15
```

Comment:
As per our discussion, I will roll this change back @mc-nv with the next batch of edits (if any).